prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,20 @@
+const languages = []
+  languages[0] = 'html'
+  languages[1] = 'css'
+  languages[2] = 'javascript'
+  languages[3] = 'react'
+  languages[4] = 'node'
+  languages[5] = 'postgres'
+console.log(languages)
+
+const firstElement = languages[0];
+console.log(firstElement);
+
+const thirdElement = languages[2];
+console.log(thirdElement);
+
+const length = languages.length;
+console.log(length);
+
+const lastElement = languages[5];
```

Comment:
instead of using the number 5 here, you want to calculate the end of the array using the length variable you defined on line 16.

Reply:
fixed the code as requested am I okay to merge?